nf_core/modules/module_test.py

```diff
@@ -56,6 +59,8 @@ def __init__(
         self.module_name = module_name
         self.no_prompts = no_prompts
         self.pytest_args = pytest_args
+        self.all_local_modules = ""
+        self.all_nfcore_modules = ""
```

Comment:
I agree with Fabian on this, I think it makes sense to just initialise them to `None`.
```suggestion
        self.all_local_modules = None
        self.all_nfcore_modules = None
```